fulp_modules/features/nanites/code/programs/utility.dm

```diff
@@ -177,8 +177,8 @@
 
 /datum/nanite_program/spreading/active_effect()
 	if(spread_cooldown < world.time)
+		spread_cooldown = world.time + 50
 		return
```

Comment:
Only adding a cooldown if its on cooldown sounds like it would just never go on cooldown, no?